behind-the-veil-siteroot/imports/ui/components/pages/request-booking/RequestBooking.jsx

```diff
@@ -292,6 +297,27 @@ const RequestBooking = () => {
         bookings: artistBookings,
     });
 
+    // find next available date in the next 2 years
+    const findNextAvailableDate = () => {
+        const currentInput = new Date(inputs.date);
+
+        for (let i = 1; i < 365*2; i++) {
```

Comment:
Where does the 2 year limit come from? I think at the moment our availability calendar extends to 3 years in the future

Reply:
@joshualoongwy updated to 3